CHANGELOG.md

```diff
@@ -33,7 +33,32 @@
 
     This way you don't have to mix your app URLs with component URLs.
 
-    Read more on [Component views and URLs](https://django-components.github.io/django-components/0.135/concepts/fundamentals/component_views_urls/).
+    Read more on [Component views and URLs](https://django-components.github.io/django-components/0.137/concepts/fundamentals/component_views_urls/).
+
+- Per-component caching - Set `Component.Cache.enabled` to `True` to enable caching for a component.
+
+    Component caching allows you to store the rendered output of a component. Next time the component is rendered
+    with the same input, the cached output is returned instead of re-rendering the component.
+
+    ```py
+    class TestComponent(Component):
+        template = "Hello"
+
+        class Cache:
+            enabled = True
+            ttl = 0.1  # .1 seconds TTL
+            cache_name = "custom_cache"
+
+            def hash_args(self, args):
+                # Custom hash method for args
+                return json.dumps(args)
+
+            def hash_kwargs(self, kwargs):
+                # Custom hash method for kwargs
+                return json.dumps(kwargs)
```

Comment:
I'm not sure I like this API. Couldn't there be a `def hash(self, args, kwargs`) method to override instead?

Maybe also change the comment to some reason why you would want to override that method.

Reply:
Good point, updated :)